lib/screens/settings/about.dart

```diff
@@ -12,87 +12,9 @@ class AboutPage extends StatelessWidget {
-                        expanded: Text(
-                            "AnitaB.org is a global nonprofit organization based in Belmont, California. Founded by computer scientists Anita Borg and Telle Whitney, the institute's primary aim is to recruit, retain, and advance women in technology."),
-                      ),
-                    ),
-                  ],
-                ),
-              ),
-              Card(
-                child: Padding(
-                  padding: const EdgeInsets.all(8.0),
-                  child: ExpandablePanel(
-                    header: Text(
-                      "What is the Mentorship System",
-                      style: TextStyle(
-                        fontWeight: FontWeight.bold,
-                        fontSize: 22,
-                      ),
-                    ),
-                    collapsed: Text("Mentorship System is a cross platform application"),
-                    expanded: Text(
-                      "Mentorship System is a cross platform application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period of time.",
-                    ),
-                  ),
-                ),
-              ),
-              Card(
-                child: Padding(
-                  padding: const EdgeInsets.all(8.0),
-                  child: ExpandablePanel(
-                    header: Text(
-                      "Contributing to Mentorship",
-                      style: TextStyle(
-                        fontWeight: FontWeight.bold,
-                        fontSize: 22,
-                      ),
-                    ),
-                    collapsed: Text("Head on over to our github to contribute"),
-                    expanded: Container(
-                      height: 40,
-                      width: 100,
-                      child: RaisedButton(
-                        color: Color.fromRGBO(36, 41, 46, 1),
-                        child: Row(
-                          children: <Widget>[
-                            Image.asset(
-                              "assets/images/GitHub-Mark-Light-120px-plus.png",
-                              width: 20,
-                              height: 25,
-                            ),
-                            SizedBox(
-                              width: 5,
-                            ),
-                            Text(
-                              "Github",
-                              style: TextStyle(color: Colors.white),
-                            ),
-                          ],
-                        ),
-                        onPressed: _launchURL,
-                      ),
-                    ),
-                  ),
-                ),
-              ),
+              _buildMainCard(context),
+              _buildGeneralCard(context),
+              _buildContriCard(context)
```

Comment:
what is `contri`?😅 Dart advises against such shortening

Reply:
@bartekpacia  done :smiley: